frontend/src/components/retailer/RetailerFlowerList.jsx

```diff
@@ -1,8 +1,21 @@
 import '../../layouts/Retailer.css'
 import { useTranslation } from 'react-i18next'
+import FlowerModal from '../FlowerModal.jsx'
+import { useState } from "react"
+import { Button, Table } from 'react-bootstrap'
 
 const RetailerFlowerList = ({ flowers }) => {
   const { t, i18n } = useTranslation()
+  const [showModal, setShowModal] = useState(false)
+  const [currentFlower, setCurrentFlower] = useState("")
+
+  const handleShow = (flower) => {
+    setShowModal(true)
+    setCurrentFlower(flower)
+  }
+  
+  const handleClose = () => setShowModal(false)
```

Comment:
`currentFlower` is left as a ghost that is not shown after `handleClose`.
It might not matter, but it is a little unclean. 

Possible fixes:
1. `handleClose` sets the current flower in end
2. `handleShow` takes in a boolean and a (possibly empty) flower and sets them to  `showModal` and `currentFlower`


This is not a deal breaker for me.

Reply:
Another Quality of Life improvement would be displaying Grower information on the flower page. These can be more fleshed out in the future now that we have a well working base for displaying information.

Reply:
Fixed the handleClose situation 👍 I think displaying Grower information can be done in the future when everything else is also more refined in this Flower View :)